ui/src/app/base/actions/space/space.test.ts

```diff
@@ -0,0 +1,66 @@
+import space from "./space";
+
+describe("space actions", () => {
+  it("should handle fetching spaces", () => {
```

Comment:
I think I only have myself to blame for this one, but all of these test descriptions don't really describe what the action creator does. They really should read "it("should return a fetch action")" or something like that, given they don't actually "handle" anything or "fetch spaces"!